✅ i66 nil class errors

kirkkwang opened the pull request:
This PR will knock out more specs failures when running in with `DEPENDENCIES_NEXT=1`.  I particularly focused on `nilClass` errors.

Ref
- https://github.com/scientist-softserv/ams/issues/66

## Failures
https://gist.github.com/kirkkwang/72ae59e8a050bedc50b0954dcfd302b8

## Failure Details
https://gist.github.com/kirkkwang/4bfe8e38bf91345894e9d6aa23039916

Commit:
✅ Fix `undefined method 'admin?' for nil:NilClass`

This commit will pass the correct scope into the `Hyrax::SearchService`
so down the stack we have access to `current_ability` which was
previously not being passed in.

Commit:
✅ Modify `SolrHelper`

This commit will modify the `SolrHelper` to correctly determine suffixes
for Solr fields.  This fixes the `csv_export_extension_spec` because it
was throwing a `nilClass` exception because the md5 method was returning
`nil`.

Commit:
✅ Skip Hyrax Batch Ingest test

This commit will skip the Hyrax Batch Ingest test until we decide if we
want to keep it or only use Bulkrax.

Ref
  - https://github.com/scientist-softserv/ams/issues/66
  - https://github.com/scientist-softserv/ams/issues/58

ShanaLMoore commented on app/helpers/solr_helper.rb:
```diff
@@ -4,7 +4,7 @@ def solr_name(field_name, *opts)
       ::Solrizer.solr_name(field_name, *opts)
     else
-      ::ActiveFedora.index_field_mapper.solr_name(field_name)
+      ::ActiveFedora.index_field_mapper.solr_name(field_name, *opts)
```
For reference: https://github.com/samvera/active_fedora/blob/99ac8d1e24bf0ee1673187ae6c0f3456ae9c04c0/lib/active_fedora/indexing/field_mapper.rb#L29

Commit:
✅ Fix specs after SolrHelper change

This commit will address the new failures that were introduced by the
`SolrHelper` change.
